content/50.discussion.md

```diff
@@ -175,7 +175,33 @@ As a result, the standard practice of requiring enrollment in a clinical trial i
+The benefits and burdens of clinical trials need to be equally distributed to ensure no influx of health disparities in the response.
+The Wealth Health Organization (WHO) Director‐General Tedros Adhanom Ghebreyesus stated his condemnation of utilizing low and middle income countries as test subjects for clinical trials, yet having highly developed countries as the majority of clinical trial representation is also not the answer [@doi:10.1002/eahr.500055].
+Figure X showcases two choropleths detailing Covid-19 clinical trial recruitment by country.
+The first choropleth only recruits from a singular country while the second choropleth recruits from multiple countries.
+In the first choropleth, China, the United States, and France are among the countries with the most clinical trial recruiting.
+However, in regards to recruiting multiple countries, only the United States and some European countries have made an effort to include other countries in clinical trials.
+In addition, many countries have little to no clinical trial recruiting, which may create inconclusivity in the data from clinical trials.
+Especially with Covid-19, equitable access to therapeutics and vaccines have been at the forefront, yet figures like the choropleths show the lack of transparency.
+Thus, singular governments might prioritize their people with vaccines, causing unequal health outcomes [@doi:10.1001/jama.2020.6641].
+Especially, with the closing of borders, keeping resources, and exiling foreigners [continue]
+ 
+When there is equal opportunity to participate in clinical trials, there is an insignificant difference in participation between racial groups [@doi:10.1002/cncr.28483].
+However, within the United States, racial minorities, such as, African-Americans, have disproportionately lower representation in clinical trials.
+Similarly, African-Americans are the most affected racial demographic by Covid-19.
+Some studies suggest communication between physicians and patients impacts whether or not a physician offers a patient participation in a clinical trial.
+The researchers utilized a linguistic analysis to assess mean word count of  phrases like voluntary participation, clinical trial, and etc.
+The data states that the mean word count of the entire visit is 1.5 more for white patients in comparisons to Black patients.
+In addition, a figure shows that the greatest disparity between white and Black patients is discussion on risks with over 2 times as many words spoken with white patients than that of Black patients [@doi:10.1002/cncr.28483].
+Thus, it is likely that Covid-19 clinical trial information is not being equally discussed with Black patients. 
+ 
+Some plausible reasons for the disparity in communication between physicians and patients could be a lack of awareness and education, mistrust in healthcare professionals, and lack of health insurance [@doi:10.1002/cncr.28483].
```

Comment:
```suggestion
```
I moved this up! I think I changed the context and made it a bit more politically aggressive, though, so let me know if you don't like that!

Reply:
it flowed really well in the new location